README.md

```diff
@@ -57,11 +57,24 @@ The entire API is `get` and `set`, and takes a range of inputs:
 >>> await plc.set('y101', True)  # Sets Y101 to true
 ```
 
+If a path is provided to a tags file when the driver is initialized, 
+tag nicknames can be used to set values and calling `.get()` without
+arguments returns all named tags.
+```python
+async with ClickPLC('the-plc-ip-address', 'path-to-tags-csv') as plc:
+        await plc.set('Tag Nickname', True)
```

Comment:
This line is a little ambiguous to me. Almost looks like something that is required to "turn on" this feature. What about using "myTagName" or similar?

Also, I'll get to this below but I thought it was going to take dict/kwargs? Maybe it does both?

Reply:
> Also, I'll get to this below but I thought it was going to take dict/kwargs? Maybe it does both?

This is precisely what I was discussing yesterday. If I want this function match the productivity in taking a dict, I have to change the function signature to kwargs and it's really weird to say that "address" is either a string or a dict of tag names and values

Reply:
That makes sense. I like the API change to match productivity but it can be a separate PR.